AnnoyancesFilter/Other/sections/annoyances.txt

```diff
@@ -1961,6 +1961,9 @@ fashion.appledaily.com.tw#%#AG_defineProperty('options.subscriptionContent', { v
 ||img.appledaily.com.tw/pay/js/minisite/paywall.js
 !+ NOT_PLATFORM(ext_safari)
 ||img.appledaily.com.tw/pay/js/paywall.*.js
+! https://github.com/AdguardTeam/AdguardFilters/issues/177376
+edhat.com##div[id^="mc4wp_form_widget-"]
```

Comment:
Please add the domain to the existing rule in subsriptions specific:
##.g1-newsletter